claragenomics/dl4atac/models/model_utils.py

```diff
@@ -124,16 +126,16 @@ def build_model(rank, interval_size, resume,
                               afunc=model_args.afunc, bn=model_args.bn)
     elif model_args.model == 'resnet':
         model = DenoisingResNet(interval_size=interval_size,
-                                afunc=model_args.afunc,
-                                bn=model_args.bn,
+                                afunc=model_args.afunc, bn=model_args.bn,
```

Comment:
let's move this back to be on two line. more readable that way

Reply:
Done.